### PR TITLE
ModelMapper removed and Transformer introduced

### PR DESCRIPTION
**Pull request by hedgehog1833**

### CODE REVIEW

**Comment by hedgehog1833** [comment]
[Remove ModelMapper](https://trello.com/c/3zDdQkwh/178-remove-modelmapper)

File: webapp/src/main/java/rocks/metaldetector/config/misc/BeanConfig.java [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-import org.modelmapper.ModelMapper;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
-@Configuration
-public class BeanConfig {
-
-  @Bean
-  public SpringApplicationContext springApplicationContext() {
-    return new SpringApplicationContext();
```
Comment (DanielW1987) [inline webapp/src/main/java/rocks/metaldetector/config/misc/BeanConfig.java:12]: maybe you should also remove the class `SpringApplicationContext`.

The base of the Metal Detector was created from a Udemy tutorial. Believe that we never important, never finished the tutorial 🤷‍♂️

Comment (hedgehog1833) [reply webapp/src/main/java/rocks/metaldetector/config/misc/BeanConfig.java:12]: Aaah funny, i didn't see that one and thought that is a class from Spring 😅

Comment (hedgehog1833) [reply webapp/src/main/java/rocks/metaldetector/config/misc/BeanConfig.java:12]: I also cleaned some stuff from the SecurityConfig

**Review by DanielW1987 — APPROVED** [review]

**Pull request merged**